parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -132,7 +132,11 @@ setupCNIDirs
 logs_to_events "AKS.CSE.installNetworkPlugin" installNetworkPlugin
 
 if [ "${IS_KRUSTLET}" == "true" ]; then
-    logs_to_events "AKS.CSE.downloadKrustlet" downloadContainerdWasmShims
+    if [[ ! -z ${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER} ]]; then
+        logs_to_events "AKS.CSE.downloadKrustlet" downloadContainerdWasmShimsFromRegistry ${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER}
```

Comment:
I'd opt for changing the event name to `downloadContainerdWasmShimsFromRegistry` so it's easier to know what's going on via kusto

Reply:
according to comment https://github.com/Azure/AgentBaker/pull/4910#discussion_r1747404793
i removed the function `downloadContainerdWasmShimsFromRegistry`